pkgs/development/ocaml-modules/resto/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitLab, buildDunePackage, uri, json-data-encoding, json-data-encoding-bson, ezjsonm, lwt }:
+
+buildDunePackage rec {
+  pname = "resto";
+  version = "0.6.1";
+  src = fetchFromGitLab {
+    owner = "nomadic-labs";
+    repo = "resto";
+    rev = "v${version}";
+    sha256 = "13h3zga7h2jhgbyda1q53szbpxcz3vvy3c51mlqk3jh9jq2wrn87";
+  };
+
+  useDune2 = true;
+
+  propagatedBuildInputs = [
+    uri
+  ];
+
+  checkInputs = [
+    json-data-encoding
+    json-data-encoding-bson
+    ezjsonm
+    lwt
+    # resto-directory
+  ];
+
```

Comment:
```suggestion
```

This dead code is probably not needed.

Reply:
Yes, I kept it because someone might figure out how to handle the circular dependencies. But we can remove it for now